src/registry_schemas/example_data/schema_data.py

```diff
@@ -1156,6 +1156,11 @@
     },
     'contactPoint': {
         'email': 'no_one@never.get'
+    },
+    'courtOrder': {
+        'fileNumber': '#1234-5678/90',
+        'orderDate': '2021-01-30T09:56:01+08:00',
+        'effectOfOrder': 'Order to disclose financial data'
```

Comment:
for now, can this be 'hasPlan', or something similar it should be open text or an enum
Also, since courtOrders can be used elsewhere, please make it a data snippet (eg BUSINESS) and include that into the alteration. it'll help the next person more.

Reply:
- I didn't understand why 'has plan of arrangement' should be an open text or an enum. Isn't it a boolean?
- Will make it a data snippet